### PR TITLE
Add python 3.9 support in github workflow.

### PR DESCRIPTION
**Pull request by menishmueli**
- Add python 3.9 to tested python versions
- Use  setup-python@v2 (v1 has bug, see https://github.com/actions/setup-python/issues/154)
- Update setup.py that the library supports python 3.9

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -76,6 +76,8 @@
         'Programming Language :: Python :: 3.5',
```
Comment (balamurugana) [inline setup.py:76]: Please remove 3.4 and 3.5

Comment (menishmueli) [reply setup.py:76]: The readme states that python 3.4 is supported, so if any the CI should also contain 3.4 and 3.5, right?
Or the readme is wrong and it works only on 3.6+?

Comment (balamurugana) [reply setup.py:76]: Please fix README.md as well. We do not support EOL'ed versions.

Comment (menishmueli) [reply setup.py]: done

**Review by balamurugana — APPROVED** [review]

**Review by kannappanr — APPROVED** [review]

**Pull request merged**